vue-components/src/components/TextArea.vue

```diff
@@ -96,6 +98,9 @@ export default defineComponent( {
 		rows: {
 			type: Number,
 			default: 2,
+			validator( value: number ): boolean {
+				return value >= 2;
```

Comment:
Is this too silly? 😅

Reply:
Nope, that looks good. I uploaded #477 with a small unit test for it so we don't accidentally break it in the future.